service/src/main/java/bio/terra/pipelines/dependencies/workspacemanager/WorkspaceClient.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.pipelines.dependencies.workspacemanager;
+
+import bio.terra.pipelines.app.configuration.external.WorkspaceServerConfiguration;
+import bio.terra.workspace.api.ControlledAzureResourceApi;
+import bio.terra.workspace.api.ResourceApi;
+import bio.terra.workspace.api.UnauthenticatedApi;
+import bio.terra.workspace.client.ApiClient;
+import org.springframework.stereotype.Component;
+
+@Component
+public class WorkspaceClient {
```

Comment:
Can we rename `Workspace` to `WorkspaceManager` in general across the board.  I feel like its more clear but happy to chat about it

Reply:
:100: